compiler/rustc_middle/src/mir/patch.rs

```diff
@@ -143,8 +142,7 @@ impl<'tcx> MirPatch<'tcx> {
         let mut delta = 0;
         let mut last_bb = START_BLOCK;
         let mut terminator_targets = Vec::new();
-        let mut inf_and_stmt: VecDeque<(SourceInfo, StatementKind<'_>)> = VecDeque::new();
-
+        let mut statements:Vec<Statement<'_>> = Vec::new();
```

Comment:
Please merge this vec with the `terminator_targets`

Reply:
Sorry I totally understood a different thing 😅